README.md

```diff
@@ -40,6 +40,7 @@ Options to use when creating pool, defaults match those used by node-pool.
 - `max_connections` - Default: `1` - Max number of connections to keep open at any given time
 - `min_connections` - Default: `0` - Min number of connections to keep open at any given time
 - `idle_timeout` - Default: `30000` - Time (ms) to wait until closing idle connections
+- `ssl` - Default: undefined - If the option is passed SSL/TLS connection will be used.
```

Comment:
Can you make this a boolean parameter instead of defined vs undefined, and then it would default false?